### PR TITLE
feat: add more precise i18next options

### PR DESCRIPTION
**Pull request by Quantumlyy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -91,9 +92,9 @@
 		"klasa-dashboard-hooks": "skyra-project/klasa-dashboard-hooks#master",
 		"node-fetch": "^2.6.1",
 		"pg": "^8.5.1",
-		"tslib": "1.11.2",
+		"tslib": "2.1.0",
```
Comment (kyranet) [inline package.json]: Revert

File: package.json [hunk 2]
```diff
@@ -91,9 +92,9 @@
...
 		"tsyringe": "^4.4.0",
-		"typeorm": "0.2.28",
+		"typeorm": "0.2.29",
```
Comment (kyranet) [inline package.json]: Revert

**Review by kyranet — CHANGES_REQUESTED** [review]

File: src/lib/util/constants.ts [hunk 3]
```diff
@@ -297,13 +297,17 @@ export const clientOptions: Partial<KlasaClientOptions> = {
...
-		i18next: {
-			preload: [...handlers.keys()],
+		i18next: (_: string[], languages: string[]) => ({
+			supportedLngs: languages,
+			preload: languages,
 			returnObjects: true,
 			returnEmptyString: false,
 			returnNull: false,
 			load: 'all',
+			lng: 'en-US',
```
Comment (kyranet) [inline src/lib/util/constants.ts:307]: Is this necessary?

Comment (Quantumlyy) [reply src/lib/util/constants.ts:307]: Not necessary but at the same time, [the docs](https://www.i18next.com/overview/configuration-options) say it only affects automatic detection.

**Review by kyranet — APPROVED** [review]

**Pull request merged**